inc/classes/class-imagify.php

```diff
@@ -470,7 +470,7 @@ private function curl_http_call( $url, $args = [] ) {
 			$url = self::API_ENDPOINT . $url;
 			$ch  = @curl_init();
 
-			if ( ! is_resource( $ch ) ) {
+			if ( ! is_resource( $ch ) && ( ! is_object( $ch ) && 'CurlHandle' !== get_class( $ch ) ) ) {
```

Comment:
Could use [`instanceof`](https://www.php.net/manual/en/language.operators.type.php) here to compress the code as it'll check if the `$ch` is an object and an instance of the class.

Reply:
@engahmeds3ed suggested that simply checking for `false` works for any php version here. Even simpler! Props!